packages/components/src/components/Select/Combobox.tsx

```diff
@@ -88,15 +94,26 @@ export const SelectCombobox = (
     });
 
     const wasClicked = useRef(false);
+
     const valueInvalid = useMemo(
-        () => inputValue && !items.find((item) => item.label.toLowerCase().includes(inputValue.toLowerCase())),
+        () => !items.find((item) => item.label.toLowerCase().includes(inputValue.toLowerCase())),
         [inputValue, items],
     );
 
+    const localStatus = useMemo(() => {
+        if (status === 'error' || valueInvalid) {
+            return 'error';
+        }
+        if (status === 'success') {
+            return 'success';
+        }
+        return 'neutral';
+    }, [status, valueInvalid]);
+
     return (
         <RadixPopover.Root open={isOpen}>
             <RadixPopover.Anchor asChild>
-                <div ref={forwardedRef} className={styles.root} data-error={valueInvalid}>
+                <div ref={forwardedRef} className={styles.root} data-status={localStatus}>
```

Comment:
```suggestion
                <div ref={forwardedRef} className={styles.root} data-status={valueInvalid || status}>
```

Reply:
this does seem simpler come to think of it 😅